toolchain/semantics/semantics_ir.h

```diff
@@ -75,7 +75,10 @@ class SemanticsIR {
                                 llvm::raw_ostream* vlog_stream) -> SemanticsIR;
 
   // Prints the full IR.
-  auto Print(llvm::raw_ostream& out) const -> void;
+  auto Print(llvm::raw_ostream& out) const -> void {
+    Print(out, /*include_builtins=*/false);
+  }
+  auto Print(llvm::raw_ostream& out, bool include_builtins) const -> void;
```

Comment:
This makes sense right now, but I might leave a TODO for the future. My guess is that when we have a prelude as an implicit import we'll want to only dump the main file, skipping at least the prelude if not all imports by default. And I'd guess that the builtins would make sense emitting as part of the prelude (and in turn, skipping from the dump when the prelude is skipped).

Anyways, clearly not a blocker.

Reply:
Adding a TODO per request.